src/base/bittorrent/torrentimpl.h

```diff
@@ -279,7 +279,9 @@ namespace BitTorrent
 
         void setAutoManaged(bool enable);
 
+        Path wantedActualPath(int index, const Path &path) const;
```

Comment:
>I wonder if we should start making use of [[nodiscard]] here and there?
Any thoughts?

I like it. But @Chocobo1 had objections to it (see [this](https://github.com/qbittorrent/qBittorrent/pull/17313#discussion_r914423083) discussion). Although I mostly disagree with his arguments, I chose not to argue, so that it would not interfere with more important things.

P.S. Please don't delete comments. It's confusing, as I get notifications about them and don't find them.

Reply:
>P.S. Please don't delete comments. It's confusing, as I get notifications about them and don't find them.

I am very sorry. I thought that I had queued my comments in pending mode. I probably hit the wrong button and then I re-reviewed this comment and deleted it.